documentation/book/ref-kafka-user.adoc

```diff
@@ -139,16 +139,17 @@ Use `\*` to allow or deny the operation from all hosts.
 The `host` field is optional and when not specified, the value `*` will be used as default.
 
 `resource`:: Specifies the resource for which does the rule apply.
-Simple Authorization supports 3 different resource types:
+Simple Authorization supports 4 different resource types:
 +
 * Topics
 * Consumer Groups
 * Clusters
+* Transactional IDs
 +
 The resource type can be specified in the `type` property.
-Use `topic` for Topics, `group` for Consumer Groups and `cluster` for clusters.
+Use `topic` for Topics, `group` for Consumer Groups, `cluster` for clusters, and `transactionId` for Transactional IDs.
```

Comment:
is that `transactionalId` and not `transactionId` ?

Reply:
Dammit ... I checked it 5 times if I have them all correctly. Thanks for noticing.